src/Nethermind/Nethermind.Facade/Proxy/Models/Simulate/SimulateBlockResult.cs

```diff
@@ -9,7 +9,7 @@
 namespace Nethermind.Facade.Proxy.Models.Simulate;
 
 public class SimulateBlockResult(Block source, bool includeFullTransactionData, ISpecProvider specProvider)
-    : BlockForRpc(source, includeFullTransactionData, specProvider)
+    : BlockForRpc(source, includeFullTransactionData, specProvider, false)
```

Comment:
Maybe better to do it by making `ShouldSerializeTotalDifficulty` virtual, rather than constructor parameter?